src/components/Sidebar/index.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <nav :class="['unnnic-sidebar', `unnnic-sidebar--${position}`]">
+    <slot
+      class="unnnic-sidebar-top"
+      name="top"
+    />
+    <ul class="unnnic-sidebar-items">
+      <li
+        v-for="(item, itemIndex) of items"
+        :key="item"
+      >
+        <SidebarItem
+          :item="item"
+          :active="{
+            item: active.itemIndex === itemIndex,
+            childIndex: active.childIndex,
+          }"
+          @navigate="handleNavigate($event)"
+        />
+      </li>
+    </ul>
+    <slot
+      class="unnnic-sidebar-bottom"
+      name="bottom"
+    />
+  </nav>
+</template>
+
+<script>
+export default {
+  name: 'UnnnicSidebar',
+};
+</script>
+
+<script setup>
+import SidebarItem from './SidebarItem.vue';
+
+const props = defineProps({
+  position: {
+    type: String,
+    default: 'left',
+    validator(value) {
+      return ['left', 'right'].includes(value);
+    },
+  },
+  width: {
+    type: String,
+    default: '200px',
+    validator(value) {
+      const pxPattern = /^\d+(\.\d+)?px$/; // numbers + px
+      const percentPattern = /^\d+(\.\d+)?%$/; // numbers + %
+      return pxPattern.test(value) || percentPattern.test(value);
+    },
+  },
+  items: {
+    type: Array,
+    required: true,
+  },
```

Comment:
To make this safer and more explanatory, you could add validations like in the [Tour component](https://github.com/weni-ai/unnnic/blob/main-vue3/src/components/Tour/propsValidation.js)